.github/workflows/apolo-release.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [apolo]
+    tags: ["v*"]
+  pull_request:
+    branches: [apolo]
+
+jobs:
+  release:
+    name: Release image
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
this will prevent `development` image push

here might be a reference 

https://github.com/neuro-inc/mlops-app-deployment/blob/ca4412caa1c364cec431c2b710322987bcc09beb/.github/workflows/ci.yaml#L19

Reply:
My understanding is that we will have an `apolo` branch which will be our default branch
And we will (for now) release development tag for an image after push, 
correct me if I am wrong, 
pushing the changes according to my understanding of the strategy

Reply:
yes, exactly, we release `development` tag by pushing into `apolo` branch, and release `latest` tag by pushing tag